src/test/groovy/org/prebid/server/functional/tests/OrtbConverterSpec.groovy

```diff
@@ -258,45 +251,48 @@ class OrtbConverterSpec extends BaseSpec {
         when: "Requesting PBS auction with ortb 2.6"
         prebidServerServiceWithNewOrtb.sendAuctionRequest(bidRequest)
 
-        then: "BidResponse shouldn contain the wlangb as on request"
+        then: "Bidder request shouldn contain the wlangb as on request"
         verifyAll(bidder.getBidderRequest(bidRequest.id)) {
             wlangb == wlangbRandomStrings
         }
     }
 
-    def "PBS should remove wlangb when we don't support ortb 2.6"() {
+    def "PBS shouldn't remove wlangb when bidder doesn't support ortb 2.6"() {
         given: "Default bid request with wlangb"
+        def wlangbRandomStrings = [PBSUtils.randomString]
         def bidRequest = BidRequest.defaultBidRequest.tap {
-            wlangb = [PBSUtils.randomString]
+            wlangb = wlangbRandomStrings
+
         }
 
         when: "Requesting PBS auction with ortb 2.5"
         prebidServerServiceWithElderOrtb.sendAuctionRequest(bidRequest)
 
-        then: "BidResponse shouldn't contain the wlangb as on request"
+        then: "Bidder request should contain the wlangb as on request"
         verifyAll(bidder.getBidderRequest(bidRequest.id)) {
-            !wlangb
+            wlangb == wlangbRandomStrings
         }
     }
 
-    def "PBS should remove device.langb when we don't support ortb 2.6"() {
+    def "PBS shouldn't remove device.langb when bidder doesn't support ortb 2.6"() {
         given: "Default bid request with device.langb"
+        def langbRandomString = PBSUtils.randomString
         def bidRequest = BidRequest.defaultBidRequest.tap {
             device = new Device().tap {
-                langb = PBSUtils.randomString
+                langb = langbRandomString
             }
         }
 
         when: "Requesting PBS auction with ortb 2.5"
         prebidServerServiceWithElderOrtb.sendAuctionRequest(bidRequest)
 
-        then: "BidResponse shouldn't contain the device.langb as on request"
+        then: "Bidder request should contain the device.langb as on request"
         verifyAll(bidder.getBidderRequest(bidRequest.id)) {
```

Comment:
You can replace `verifyAll` with `assert` due to a single check. Same for other places